File: Ejercicio5.py
Arreglo = [150, 250, 350, 450, 550]
Continuar = "S"

Valor_Encontrar = None

Encontrado = None

while Continuar.upper() == "S":
    Valor_Encontrar = int(input("Ingrese Valor a buscar "))
    Encontrado = False

    for i in range(len(Arreglo)):
        if Arreglo[i] == Valor_Encontrar:
            print("El Valor Se ingresado se encuentra en la posicion: ", i)
            Encontrado = True

    if not Encontrado:
     print( "El valor ingresado no se encuentra en el arreglo.")

    Continuar = input("Desea probar con otro valor? (S/N)")

File: Ejercicio5.java
import java.util.Scanner;

public class Ejercicio5 {
    public static void main(String[] args) {
        int[] Arreglo = {100, 200, 300, 400, 500};
        char Continuar = 'S';
        int Valor_a_Buscar;
        boolean encontrado;

        Scanner scanner = new Scanner(System.in);

        do {
            System.out.println("Ingrese valor a buscar: ");
            Valor_a_Buscar = scanner.nextInt();

            encontrado = false;

            for (int i = 0; i < Arreglo.length; i++) {
                if (Arreglo[i] == Valor_a_Buscar) {
                    System.out.println("El valor ingresado se encuentra en la posición: " + i);
                    encontrado = true;
                }
            }

            if (!encontrado) {
                System.out.println("El valor ingresado no se encuentra en el arreglo.");
            }

            System.out.print("¿Desea probar con otro valor? (S/N): ");
            Continuar = scanner.next().toUpperCase().charAt(0);

        } while (Continuar == 'S');

        scanner.close();
    }
}
